adapters/appnexus/appnexus_test.go

```diff
@@ -24,10 +25,14 @@ func TestJsonSamples(t *testing.T) {
 }
 
 func TestMemberQueryParam(t *testing.T) {
-	uriWithMember := appendMemberId("http://ib.adnxs.com/openrtb2?query_param=true", "102")
-	expected := "http://ib.adnxs.com/openrtb2?query_param=true&member_id=102"
-	if uriWithMember != expected {
-		t.Errorf("appendMemberId() failed on URI with query string. Expected %s, got %s", expected, uriWithMember)
+	uri, err := url.Parse("http://ib.adnxs.com/openrtb2?query_param=true")
```

Comment:
**Nitpick:** Consider modifying this function to use `assert` for assertions and rename the function to `TestAppendMemberID`. As a rule of thumb I suggest we name test functions as actual function name it's testing with the `Test` prefix. I understand your PR isn't the one that added this test function so this is not to blame you :)

```
func TestAppendMemberID(t *testing.T) {
	uri, err := url.Parse("http://ib.adnxs.com/openrtb2?query_param=true")
	assert.NoError(t, err, "Failed to parse URI with query string")
	uriWithMember := appendMemberId(*uri, "102")
	expected := "http://ib.adnxs.com/openrtb2?member_id=102&query_param=true"
	assert.Equal(t, expected, uriWithMember.String(), "Failed to append member id to URI with query string")
}
```

Reply:
updated the name and assertions in [040e2fb](https://github.com/prebid/prebid-server/pull/2747/commits/040e2fbd75507b7ffaa79d8a17f23f5e2148ecfc)